plotter/histogram_test.go

```diff
@@ -62,3 +63,33 @@ func ExampleHistogram() {
 func TestHistogram(t *testing.T) {
 	cmpimg.CheckPlot(ExampleHistogram, t, "histogram.png")
 }
+
+func TestSingletonHistogram(t *testing.T) {
+	done := make(chan struct{}, 1)
+	go func() {
+		defer close(done)
+		p, err := plot.New()
+		if err != nil {
+			t.Errorf("unexpected error from plot.New: %v", err)
+		}
+
+		hist, err := NewHist(Values([]float64{1.0}), 60)
```

Comment:
I suspect the same problem would be triggered by many equal values, consider adding a test to make sure the fix solves that case too?